src/lib/merkle_ledger/test/test.ml

```diff
@@ -0,0 +1,16 @@
+(* Testing
+   -------
+
+   Component: Merkle ledger tests
+   Subject: Run all Merkle ledger tests
+   Invocation: dune exec src/lib/merkle_ledger/test/test.exe
+*)
+
+open Merkle_ledger_tests
+
+let () =
```

Comment:
Can we split in different executable, please?
You can use the stanza `tests` with the field `names` instead, for multiple test. It will also be run in parallel locally.

Reply:
Were you thinking of having separate test runner files like `test_converting_runner.ml` with the content

```
open Merkle_ledger_tests

let () = Alcotest.run "In-memory converting db" Test_converting.tests
```

I could also put all the existing files like `test_converting.ml` in a `test/helpers/` directory and have the top-level `test_converting.ml` just be a runner for the test.

Reply:
The issue I found is that the current structure allows the `test_*` modules to be exposed as libraries, but if I make them `tests` then they can't simultaneously be libraries because of how dune works. Some other ledger tests (the syncable ledger tests in a different package, and some tests within this package) reuse some of the test definitions, so that shared code does need to be exposed in some way. I could look into pulling out just the shared code into some libraries as well, to be a bit more discriminating in what gets exposed to other packages.